post_installation_configuration/configuring-multi-arch-compute-machines/creating-multi-arch-compute-nodes-aws.adoc

```diff
@@ -6,13 +6,14 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-To create an AWS cluster with multi-architecture compute machines, you must first create a single-architecture AWS installer-provisioned cluster with the multi-architecture installer binary. For more information on AWS installations, refer to xref:../../installing/installing_aws/ipi/installing-aws-customizations.adoc#installing-aws-customizations[Installing a cluster on AWS with customizations]. You can then add a ARM64 compute machine set to your AWS cluster.
+To create an AWS cluster with multi-architecture compute machines, you must first create a single-architecture AWS installer-provisioned cluster with the multi-architecture installer binary. For more information on AWS installations, see xref:../../installing/installing_aws/ipi/installing-aws-customizations.adoc#installing-aws-customizations[Installing a cluster on AWS with customizations]. You can then add a 64-bit ARM or 64-bit x86 compute machine set to your AWS cluster.
```

Comment:
```suggestion
To create an AWS cluster with multi-architecture compute machines, you must first create a single-architecture AWS installer-provisioned cluster with the multi-architecture installer binary or migrate your existing cluster to the multi-architecture payload. For more information on AWS installations, see xref:../../installing/installing_aws/ipi/installing-aws-customizations.adoc#installing-aws-customizations[Installing a cluster on AWS with customizations]. You can then add a 64-bit ARM or 64-bit x86 compute machine set to your AWS cluster.
```

[Slightly OT] Should we clarify this?

Reply:
Makes sense.
Updated as per the suggestion.